### PR TITLE
Fix Rubocop Layout violations - 2

### PR DESCRIPTION
**Pull request by macanudo527**
#### What? Why?

- Part of #6055 

This fixes 5 more rubocop rules that are violated in our codebase. It was generated automatically with the [rubocop-autocorrect.sh](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/script/rubocop-autocorrect.sh) script.

#### What should we test?

No testing required since these are safe corrections

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: spec/system/admin/order_cycles/complex_updating_specific_time_spec.rb
Comment (mkllnk) [inline spec/system/admin/order_cycles/complex_updating_specific_time_spec.rb:84]: This is interesting. Does this mean that the string is not seen as method argument? Oh, I can't test it because the whole spec is disabled. This is dead and broken code. Apparently this spec was commented out as flaky three years ago:

* 042b4faf1e05fa73ba72886d23b44fae1fcf9ab2

@filipefurtad0 Can you asses if we should fix the spec or delete it?

File: spec/system/admin/order_cycles/simple_spec.rb
Comment (mkllnk) [inline spec/system/admin/order_cycles/simple_spec.rb:784]: I confirmed for this spec that the string was ignored and not passed on as an argument. Adding parenthesis fixes the code. I will add a commit.

File: spec/system/admin/adjustments_spec.rb
Comment (mkllnk) [inline spec/system/admin/adjustments_spec.rb:71]: It would probably be nicer to have all the values in one line.

**Review by mkllnk — APPROVED** [review]
Nice work. I like it when an automated tool highlights a mistake in the code.

**Review by rioug — APPROVED** [review]
Nice one !

**Comment by rioug** [comment]
Waiting on Filipe's feedback before merging

File: spec/system/admin/order_cycles/complex_updating_specific_time_spec.rb
Comment (filipefurtad0) [reply spec/system/admin/order_cycles/complex_updating_specific_time_spec.rb]: Nice, thanks for digging that out. I would keep it and create an issue to keep track of it - I did so [here](https://github.com/openfoodfoundation/openfoodnetwork/issues/11266).

I've tried to fix it, but it's not a quick win, it's difficult to get a consistent behavior in the OC pages (the `add Distributor` button...)... Anyway, out of scope here.

**Comment by filipefurtad0** [comment]
> Waiting on Filipe's feedback before merging

Good to merge on my end :+1:

**Pull request merged**
